types/types.go

```diff
@@ -448,7 +448,7 @@ const (
 // OptionalBoolFalse.  The function is meant to avoid boilerplate code of users.
 func NewOptionalBool(b bool) OptionalBool {
 	o := OptionalBoolFalse
-	if b == true {
+	if b {
```

Comment:
(I’m not thrilled about this one, but, oh well, I can see how the linter does not like the previous version.)